js/ui/scheduler/ui.scheduler.header.js

```diff
@@ -173,7 +173,8 @@ var SchedulerHeader = Widget.inherit({
             },
             selectedItem: this.option("currentView"),
             tabIndex: this.option("tabIndex"),
-            focusStateEnabled: this.option("focusStateEnabled")
+            focusStateEnabled: this.option("focusStateEnabled"),
+            showNavButtons: true
```

Comment:
It is the very strange change

Reply:
Why? We need navigation buttons, but showNavButtons is 'false' by default for some themes (material)

Reply:
I think set the option to true here can change behavior on mobile devices

Reply:
I agree. I'll remove this after we select the right default combination of scrollByContent and showNavButtons options for dxTabs in material theme.